packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -87,9 +110,16 @@ const drawCommits = (svg, commits, modifyGraph) => {
   const sortedKeys = keys.sort((a, b) => {
     return commits[a].seq - commits[b].seq;
   });
+
+  const isParallelCommits = getConfig().gitGraph.parallelCommits;
   sortedKeys.forEach((key) => {
     const commit = commits[key];
 
+    if (isParallelCommits && commit.parents.length) {
+      const closestParent = findClosestParent(commit.parents, commits);
+      pos = dir === 'TB' ? commitPos[closestParent].y + 40 : commitPos[closestParent].x + 40;
```

Comment:
There is no check for the case when `closesParent` is blank string. I am not sure, maybe there is always an empty element in `commitPos`, but it seems strange a little bit. Initial commit should not have a parent, for instance, if I am not mistaken

Reply:
`closestParent` will never be blank here I believe. The `commit.parents.length` in the `if` condition garantees that the commit has atleast 1 parent, so the `findClosestParent` inside always returns a parent. Initial commit for example won't satisfy this `if` condition since its `commit.parents.length` will be 0.